src/api-doc/api-doc.yaml

```diff
@@ -3387,15 +3429,20 @@ paths:
                       correlation: c1987eb0-c58a-415c-99bc-a9de62518318
                       meeting_platform: BBB
   /mentoring/v1/questions/update/{QuestionId}:
-    patch:
```

Comment:
@sumanvpacewisdom why this method changed, did you verified ?

Reply:
It is changed int the postman collection and not updated in the api doc